frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/utils.ts

```diff
@@ -0,0 +1,380 @@
+import { Tissue } from "src/views/WheresMyGene/common/types";
```

Comment:
this file has all the SVG rendering functions. No business logics changed here beside a few tiny TypeScript changes to remove the usage of assertion `!`